File: java/sprint3/K/Solution.java
import java.util.Arrays;

public class Solution {
	public static int[] merge(int[] arr, int left, int mid, int right) {
		// Your code
		// “ヽ(´▽｀)ノ”	
		return null;
	}

	public static void merge_sort(int[] arr, int left, int right) {
		// Your code
		// “ヽ(´▽｀)ノ”
	}

	public static void main(String[] args) {
		int[] a = {1, 4, 9, 2, 10, 11};
		int[] b = merge(a, 0, 3, 6);
		int[] expected = {1, 2, 4, 9, 10, 11};
		assert Arrays.equals(b, expected);
		int[] c = {1, 4, 2, 10, 1, 2};
		merge_sort(c, 0, 6);
		int[] expected2 = {1, 1, 2, 2, 4, 10};
		assert Arrays.equals(c, expected2);
	}
}

File: swift/sprint3/K/solution.swift

func merge_sort(arr: inout [Int]) {
    // Your code
    // ヽ(´▽`)/
}

func merge(a: [Int], b: [Int]) -> [Int] {
    // Your code
    // ヽ(´▽`)/
    return [1, 3, 4]
}

func test () {
    var a: [Int] = [1, 4, 9]
    var b: [Int] = [2, 10, 11]
    var c = merge(a: a, b: b);
    var expected: [Int] = [1, 2, 4, 9, 10, 11];
    assert(c == expected);
    var d: [Int] = [1, 4, 2, 10, 1, 2]
    var sorted = [1, 1, 2, 2, 4, 10]
    merge_sort(arr: &d);
    assert(d == sorted);
}
